### ISSUE TITLE
Set default/initial rule and award

### ISSUE THREAD

**Issue by bjarnef** [TeaCommerce/Tea-Commerce-for-Umbraco#26]
Set default/initial rule and award otherwise when it's blank and you click "add", nothing happens, because there is an error:

```
TypeError: Cannot read property 'alias' of undefined
```

so maybe just set the first element in the dropdownlist as default value?

**Comment by andersburla** [comment]
I think its better to have the shop owner select which rule to add. Don't you think so? From a UX perspective.

**Comment by bjarnef** [comment]
Yes, then it should just have a default value like "choose" or "select" and validate against it .. I think this issue was that a null value was submitted.
